### PR TITLE
ZENKO-2820 List objects: first iteration

### PR DESCRIPTION
**Pull request by nicolas2bert**
Screen-Recording-2020-09-22-at-10 05 08-AM

### CODE REVIEW

File: src/react/databrowser/objects/ObjectList.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        columns,
+        data: objects,
+        disableSortRemove: true,
+        autoResetFilters: false,
+        autoResetSortBy: false,
+    }, useFilters, useSortBy);
+
+    return <L.ListSection>
+        <T.SearchContainer>
+            <T.ExtraButton icon={<i className="fas fa-plus" />} text="Upload" variant='info' onClick={() => dispatch(push(`/buckets/${bucketNameParam}/upload-object`))} size="default" type="submit" />
```
Comment (terrydervaux) [inline src/react/databrowser/objects/ObjectList.jsx:81]:
```suggestion
            <T.ExtraButton icon={<i className="fas fa-plus" />} text="Upload Object" variant='info' onClick={() => dispatch(push(`/buckets/${bucketNameParam}/upload-object`))} size="default" type="submit" />
```

**Review by anurag4DSB — APPROVED** [review]
LGTM

**Review by terrydervaux — APPROVED** [review]
LGTM.

Nice to have:
- align the `Size` column content on the right.
- change the upload button text.

**Comment by nicolas2bert** [comment]
@bert-e approve

**Comment by nicolas2bert** [comment]
@bert-e approve

**Comment by nicolas2bert** [comment]
@bert-e approve

File: src/react/databrowser/objects/ObjectList.jsx (hunk 1 above)
Comment (nicolas2bert) [reply src/react/databrowser/objects/ObjectList.jsx:81]: Thank you :) I added it to the second iteration ticket: https://scality.atlassian.net/browse/ZENKO-2847

**Pull request merged**